modules/ROOT/pages/intro-mule-message.adoc

```diff
@@ -75,7 +75,8 @@ In Mule 3, Mule connectors and transports that need to send additional data, suc
 The example above performs an HTTP request in which individual DataWeave scripts generate headers and query parameters without the need to set message properties and without generating any side effects on the message.
 
 The same concept can be applied for message sources that emits responses, such an HTTP listener:
-
+//TODO: ANA OR SOME OTHER KIND MULE DEV :) , COULD YOU PLEASE HELP WITH THE vars.outboundHeaders default {}. 
+//IS THE KEYWORD default correct here? My understanding is that you set default differently in DW 2.0.
```

Comment:
This example is correct. I believe default might be a function and hence the confusion.